.github/workflows/upload-nightly-release.yaml

```
@@ -0,0 +1,47 @@
+name: Build nightly PennyLane releases for TestPyPI
+
+on:
+  schedule:
+    # Run every weekday at 23:40 EDT (cron is in UTC)
+    - cron: "40 3 * * 2-6"
```

Spread out scheduled actions:
```suggestion
    - cron: "20 3 * * 2-6"
```